### DISCUSSION TITLE
RESET WINDOWS TO FACTORY

### DISCUSSION THREAD

**Discussion by Daxer97** [simeononsecurity/Windows-Optimize-Harden-Debloat#61] [Q&A] [answered]
It is possible to revert to the factory setting after lunching the script ? Moreover, it is possible to undo the changes ive made with this script ? How ?

The system does not allow me to return to factory

Thanks

**Comment by simeononsecurity** [comment, marked as answer]
> It is possible to revert to the factory setting after lunching the script ? Moreover, it is possible to undo the changes ive made with this script ? How ?
> 
> The system does not allow me to return to factory
> 
> Thanks

Hello @Daxer97 
Please take this as a lesson to read the [readme](https://github.com/simeononsecurity/Windows-Optimize-Harden-Debloat/blob/master/README.md) of the repo before running the script. 

This script does implement a restore point prior to running the script. If you are still able to sign in as an administrator, you should be able to follow the guides below.
https://www.howtogeek.com/220/use-system-restore-on-windows/
https://support.microsoft.com/en-us/windows/use-system-restore-a5ae3ed9-07c4-fd56-45ee-096777ecd14e

If you're not able to sign in, you'll have to follow this guide to reinstall windows from scratch:
https://simeononsecurity.ch/articles/how-to-install-windows-10-and-windows-11-from-scratch/

If you're having any other issues in particular, we'd be glad to help you. Please just give us updates here.
